docs-source/content/userguide/introduction/architecture.md

```diff
@@ -88,3 +89,8 @@ The team developing the operator felt that these considerations provided adequat
 ### Network name predictability
 
 The operator uses services to provide stable, well-known DNS names for each server.  These names are known in advance of starting up a pod to run a server, and are used in the `ListenAddress` fields in the WebLogic Server configuration to ensure that servers will always be able to find each other.  This also eliminates the need for pod names or the actual WebLogic Server instance names to be the same as the DNS addresses.
+
+### Draining a node and PodDisruptionBudget
+A Kubernetes cluster administrator can [drain a Node](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) for repair, upgrade, or scaling down the Kubernetes cluster.
+
+Beginning in version 3.2, the operator takes advantage of the [PodDisruptionBudget](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets) feature offered by Kubernetes for high availability during a Node drain operation. The operator creates a PodDisruptionBudget (PDB) for each WebLogic cluster in the Domain namespace to limit the number of WebLogic Server pods simultaneously evicted when draining a node. The maximum number of WebLogic cluster's server pods evicted simultaneously is determined by the `maxUnavailable` field on the Domain resource. The `.spec.minAvailable` field of the PDB is calculated as `domain.spec.replicas - domain.spec.maxUnavailable`. For example, if you have a WebLogic cluster with three replicas and a `maxUnavailable` of 1, the `.spec.minAvailable` for PDB is set to 2. In this case, Kubernetes ensures that at least two pods for the WebLogic cluster's Managed Servers are available at any given time, and it only evicts a pod when all three pods are ready. For details about safely draining a node and the PodDisruptionBudget concept, see [Safely Drain a Node](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) and [PodDisruptionBudget](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/).
```

Comment:
"The `.spec.minAvailable` field of the PDB is calculated as `domain.spec.replicas - domain.spec.maxUnavailable`."

Do we consider the `replicas` and `maxUnavailable` values in the `Cluster` level?

Reply:
Yes, we do consider `replicas` and `maxUnavailable` values at the cluster level. I'm currently using the same `maxUnavailable ` value that's used for rolling restarts. Since this is the architecture document, I didn't want to include too much details here. 
@rjeberhard, @alai8  - Do you have any suggestions on whether we should describe `maxUnavailable` setting in more detail here or in some other document? Currently, it's documented in https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/#rolling-restarts. Thanks.

Reply:
Probably just need to rephrase the sentence here that does not give an impression that only domain level configuration is being considered in the calculation, something like "The .spec.minAvailable field of the PDB for a cluster is calculated from the difference of the current replicas count and maxUnavailable value configured for the cluster"